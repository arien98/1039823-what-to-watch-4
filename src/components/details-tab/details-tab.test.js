import React from "react";
import renderer from "react-test-renderer";
import DetailsTab from "./details-tab.jsx";

describe(`main renders correctly`, () => {
  test(`it renders correctly`, () => {
    const props = {
      starring: [`Anthony Mann`, `Andy Dark`],
      director: `Guy Ritchie`,
      runtime: 30,
      genre: `Drama`,
      releaseDate: 1980
    };

    const tree = renderer
      .create(<DetailsTab {...props} />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
