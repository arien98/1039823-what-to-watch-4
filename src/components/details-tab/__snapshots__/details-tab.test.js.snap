// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main renders correctly it renders correctly 1`] = `
<div
  className="movie-card__text movie-card__row"
>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Director
      </strong>
      <span
        className="movie-card__details-value"
      >
        Guy Ritchie
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Starring
      </strong>
      <span
        className="movie-card__details-value"
      >
        Anthony Mann, 
Andy Dark
      </span>
    </p>
  </div>
  <div
    className="movie-card__text-col"
  >
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Run Time
      </strong>
      <span
        className="movie-card__details-value"
      >
        0h 30m
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Genre
      </strong>
      <span
        className="movie-card__details-value"
      >
        Drama
      </span>
    </p>
    <p
      className="movie-card__details-item"
    >
      <strong
        className="movie-card__details-name"
      >
        Released
      </strong>
      <span
        className="movie-card__details-value"
      >
        1980
      </span>
    </p>
  </div>
</div>
`;
